packages/cli-test/src/cli/commands/platform.ts

```diff
@@ -203,6 +203,13 @@ export default {
     return new Promise((resolve, reject) => {
       // kill the shell process
       shell.kill(proc).then(() => {
+
+        // For the team being, we don't wait for the SLACK_TRACE_PLATFORM_RUN_STOP trace on Windows
```

Comment:
I find it's good to describe _why_ we are not waiting; that "why" tends to be more helpful in the far future after everyone involved has long forgotten why the decision was made 😅 

```suggestion
        // Due to the complexity of gracefully shutting down processes on Windows / lack of interrupt signal support, we don't wait for the SLACK_TRACE_PLATFORM_RUN_STOP trace on Windows
```

Reply:
Fair 😛